src/server/pfs/server/server_test.go

```diff
@@ -2984,6 +2984,14 @@ func TestGlob(t *testing.T) {
 	fileInfos, err = c.GlobFile(repo, "master", "dir1/*")
 	require.NoError(t, err)
 	require.Equal(t, numFiles, len(fileInfos))
+	fileInfos, err = c.GlobFile(repo, "master", "/non-existant-glob*")
```

Comment:
Existent not existant. Minor nit pick lol.

Reply:
Done.